apps/prairielearn/src/pages/instructorAssessmentAccessOverrides/instructorAssessmentAccessOverrides.js

```diff
@@ -0,0 +1,152 @@
+// @ts-check
+import * as express from 'express';
+const asyncHandler = require('express-async-handler');
+import * as sqldb from '@prairielearn/postgres';
+import * as error from '@prairielearn/error';
+import { getEnrollmentForUserInCourseInstance } from '../../models/enrollment';
+import { selectUserByUid } from '../../models/user';
+
+const router = express.Router();
+const sql = sqldb.loadSqlEquiv(__filename);
+
+async function getUserIdAndCheckEnrollment({ uid, course_instance_id }) {
+  const user = await selectUserByUid(uid);
+  if (!user) {
+    throw error.make(400, `User ${uid} does not exist.`);
+  }
+
+  const enrollment = await getEnrollmentForUserInCourseInstance({
+    user_id: user.user_id,
+    course_instance_id: course_instance_id,
+  });
+  if (!enrollment) {
+    throw error.make(400, `User ${uid} is not enrolled in this course instance.`);
+  }
+
+  return user.user_id;
+}
+
+router.get(
+  '/',
+  asyncHandler(async (req, res) => {
+    const result = await sqldb.queryAsync(sql.select_assessment_access_policies, {
+      assessment_id: res.locals.assessment.id,
+      student_uid: req.body.student_uid,
+      course_instance_id: res.locals.course ? res.locals.course_instance.id : null,
+      course_id: res.locals.course ? res.locals.course.id : null,
```

Comment:
These should no longer be needed once you remove them from the query (in fact, `course_instance_id` was already removed).

Reply:
fixed in [92cbd0c](https://github.com/PrairieLearn/PrairieLearn/pull/9103/commits/92cbd0c103797f07ae80ace8887d7ca4ae0f306a)